scipy/stats/stats.py

```diff
@@ -1039,6 +1095,16 @@ def moment(a, moment=1, axis=0):
 
     """
     a, axis = _chk_asarray(a, axis)
+
+    contains_nan, nan_policy = _contains_nan(a, nan_policy)
+
+    # TODO: implement the delegation to mstats when nan_policy == 'omit'
+    # if contains_nan and nan_policy == 'omit':
+    #     a = ma.masked_invalid(a)
```

Comment:
From ``$ grin "= stats." mstats_basic.py`` you can see that ``stats`` is hardly used within ``mstats`` except for copying some docstrings. So the circular imports can be broken like so I think:

- create a new file ``mstats_add_docs.py`` which imports ``mstats_basic`` and ``stats``
- import ``mstats_add_docs`` in ``mstats``; don't import ``mstats_basic`` there
- import ``mstats_basic`` in ``stats.py`` and use it to forward the ``nan_policy='omit'`` cases.
- remove ``from . import stats`` from ``mstats_basic.py``